modernizer-maven-plugin/src/main/java/org/gaul/modernizer_maven_plugin/ModernizerMojo.java

```diff
@@ -274,19 +291,19 @@ public void execute() throws MojoExecutionException {
                 is = new FileInputStream(exclusionsFilePath);
             } else {
                 is = this.getClass().getClassLoader().getResourceAsStream(
-                        exclusionsFilePath);
+                    exclusionsFilePath);
             }
             if (is == null) {
                 throw new MojoExecutionException(
-                        "Could not find exclusion file: " +
```

Comment:
revert this please

Reply:
Changed it to whatever is in the original plugin.

Reply:
You deleted the word `exclusion`: https://github.com/gaul/modernizer-maven-plugin/blob/589ca71078fd0d90ce3f8960142c8be07f9a9c2d/src/main/java/org/gaul/modernizer_maven_plugin/ModernizerMojo.java#L281